sample-btcd.conf

```diff
@@ -199,6 +199,13 @@
 ; the default).
 ; notls=1
 
+; ------------------------------------------------------------------------------
+; Optional Transaction Indexes
+; ------------------------------------------------------------------------------
+; Build and maintain a full address-based transaction index.
+; addrindex=1
+; Delete the entire address index on start up, then exit.
```

Comment:
Nitpick: All of the other options have a blank line between them with blank lines separating the groupings.  Should probably be consistent here.